api/controllers/storyText.js

```diff
@@ -0,0 +1,16 @@
+const generateStory = require("../clients/GPTclient");
+const GPT_prompt_gen = require("../prompts/GPTPromptGen")
+
+const StoryController = {
+  Index: async (req, res) => {
+    try {
+      const prompts = GPT_prompt_gen(req.body.userinput)
+      const story = generateStory(prompts)
+      res.status(200).json({ storyText: story });
+    } catch (error) {
+      res.status(error.status).json({ message: error.message });
```

Comment:
this currently wont work as the client itself is not throwing any errors at the moment (need to throw errors and assign codes to it)